### PR TITLE
Remove edit route

### LINKED ISSUE #106: doctor-name/edit pages may get indexed
Either
- remove a special /edit URL for editing (so the url stays doctor-name)
or 
- add a meta tag to header:
`<meta name="robots" content="noindex">`

Watch out for cases with edit in doctor's name, like 
https://stage-zdravniki.sledilnik.org/sl/gp/aladic-edit

### PR DESCRIPTION
**Pull request by mitjapotocin**
Fixes #106

Fixes also: When when after editing and posting doctor info if you clicked again on 'Prijavi napako v objavljenih podatkih' you for 404.

### CODE REVIEW

**Comment by lukarenko** [comment]
This is great and works fine!

**Comment by mitjapotocin** [comment]
Will merge after @mihaerzen merges #81, so he will have fewer merge conflicts.

**Review by jalezi — APPROVED** [review]

**Pull request merged**
